### ISSUE TITLE
Bottom view Scrolls to Top when tableview reloads and row height changed

### ISSUE THREAD

**Issue by mohanthakur92** [OfTheWolf/TwitterProfile#19]
I am using this library and i am using tableview in the bottomView i am having expand and collapse functionality when i expand and reload table the bottom view scroll ups and starts from top.
Any solution is appreciated

**Comment by mohanthakur92** [comment]
@OfTheWolf can you please help me in this?

**Comment by OfTheWolf** [comment]
@mohanthakur92 Can you share a minimum sample project to replicate this please?

**Comment by mohanthakur92** [comment]
ok i will share the sample code. Basically in the bottom tabs i used tableview and when i reload tableview and scrolls it tableview moves to top (its insets changes)

**Comment by hstdt** [comment]
I believe it's caused by `updateOverlayScrollContentSize` in `ContainerViewController`, but my fix still not perfect.

bottom tableview/scrollview content size changed(estimatedRowHeight  tableView when scrolls to bottom, or load more) --> overlayScrollView content size change -->  overlayScrollView scrolls --> bottom tableview/scrollview scrolls(**here causes Scrolls to Top** even if revert contentOffset.Y, and drag scroll gesture will be interrupted)

<img width="1340" alt="CleanShot 2022-05-10 at 00 11 40" src="https://user-images.githubusercontent.com/10215098/167452485-3fad7c95-f17e-42b2-a476-0040700c7bd3.png">

```swift
    private func updateOverlayScrollContentSize(with bottomView: UIView) {
        let contentOffsetY = overlayScrollView.contentOffset.y
        self.overlayScrollView.contentSize = getContentSize(for: bottomView)
        if contentOffsetY > 0 {
            // revert contentOffsetY after contentSize changes, but still not perfect
            self.overlayScrollView.setContentOffset(.init(x: 0, y: contentOffsetY), animated: false)
        }
    }
```

**Comment by BH102** [comment]
@hstdt @mohanthakur92 I'm having same issue, did you guys solve it?

**Comment by BH102** [comment]
When I use table view with static cell size everything works fine but when I use table view with dynamic cell size, the scrolling is lagging specially when scrolling up

**Comment by hstdt** [comment]
> @hstdt @mohanthakur92 I'm having same issue, did you guys solve it?

No, you can try this one. https://github.com/pujiaxin33/JXPagingView

**Comment by BH102** [comment]
@hstdt Thank you, I've used it it's great

**Comment by BH102** [comment]
> > @hstdt @mohanthakur92 I'm having same issue, did you guys solve it?
> 
> No, you can try this one. https://github.com/pujiaxin33/JXPagingView



> > @hstdt @mohanthakur92 I'm having same issue, did you guys solve it?
> 
> No, you can try this one. https://github.com/pujiaxin33/JXPagingView


Thank you, I've used it it's great
